### ISSUE TITLE
Fresh Install Not Working

### ISSUE THREAD

**Issue by darrencope** [jaroslawhartman/withings-sync#29]
Hi,

When doing pip install withings-sync, I get the following error:

> Could not find a version that satisfies the requirement requestscloudscraper (from withings-sync) (from versions: )
> No matching distribution found for requestscloudscraper (from withings-sync)

Admittedly new at this, so hopefully I'm not doing something wrong, but from the changelog it looks like you have been working on cloudscraper stuff recently, so I thought this may be relevant.

**Comment by seanhoughton** [comment]
I getting the same thing with 3.2.0 even though that version should have the fix. Here's the change: https://github.com/jaroslawhartman/withings-sync/commit/e62bf29703d7516cc4d4fa292250148f05a0e35a

**Comment by gmiossec29** [comment]
You can also install with pip from github : 
`pip install git+https://github.com/jaroslawhartman/withings-sync.git`

**Comment by sfregosi** [comment]
@gmiossec29 Thank you! This fixed my issue #28

**Comment by darrencope** [comment]
Thanks guys. I'm still not having any luck, but for a different reason now I think.

I'm installing on a Raspberry Pi, and it's crashing at the 'Running setup.py bdist_wheel for lxml..." stage. I see from https://stackoverflow.com/questions/33064433/lxml-will-never-finish-building-on-ubuntu that it's likely an out of memory issue, and will try to install the pre-built binaries once my RPi comes back to life.

**Comment by darrencope** [comment]
I was able to install the pre-built binaries of lxml and then the withings-sync setup installed successfully. However, I cannot execute it without a 'command not found' error. I hope I am missing something simple, but it seems like the install didn't actually work. Any ideas?

**Comment by midzelis** [comment]
I have this same issue 
```
root@CT100:~# pip install withings-sync
Collecting withings-sync
  Downloading withings-sync-3.2.0.tar.gz (17 kB)
Collecting lxml
  Downloading lxml-4.6.3-cp38-cp38-manylinux2014_x86_64.whl (6.8 MB)
     |████████████████████████████████| 6.8 MB 2.1 MB/s 
ERROR: Could not find a version that satisfies the requirement requestscloudscraper (from withings-sync) (from versions: none)
ERROR: No matching distribution found for requestscloudscraper (from withings-sync)```

**Comment by stv0g** [comment]
Fixed by e62bf29
